src/nft/hooks/useDebouncedCallback.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useRef } from 'react'
```

Comment:
Add this to src/hooks/useDebounce or move it to src/hooks/useDebouncedCallback, as it is a general utility.